### PR TITLE
Add index pages to some sidebar categories

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/933

This PR is conservative in where we add index pages. I want to try it out on some menus first, and if we like it there then we can try it on other menus. To me, it made the most sense to add index pages for all menus that contain different cloud docs. This way, we can link to the index page instead of having to list 3 links for each cloud whenever we mention those details in our docs.

### CODE REVIEW

File: sidebarsAstro.js [hunk 1]
```diff
@@ -84,6 +84,11 @@ module.exports = {
         {
           type: 'category',
           label: 'Install Astro',
+          link: {
+            type:'generated-index',
+            title: 'Install Astro',
+            description: 'Install the Astro platform on your cloud.'
```
Comment (kwcanuck) [inline sidebarsAstro.js:90]:
```suggestion
            description: 'Install the Astro platform in your cloud.'
```

File: sidebarsAstro.js [hunk 2]
```diff
@@ -129,6 +134,12 @@ module.exports = {
         {
           type: 'category',
           label: 'Cloud configuration reference',
+          link: {
+            type:'generated-index',
+            title: 'Cloud configuration reference',
+            description: 'Learn about all default and supported Astro cloud configurations.'
```
Comment (kwcanuck) [inline sidebarsAstro.js:140]:
```suggestion
            description: 'Learn about the default and supported Astro cloud configurations.'
```

**Review by kwcanuck — APPROVED** [review]

**Comment by jwitz** [comment]
@paolaperaza @kwcanuck This is ready for review again, but on a different front: I was able to load custom properties from the docs so that they use custom icons in their index cards! I imagine that we'll be doing this for all cloud-defined index menus in the future. 

Screen Shot 2022-08-24 at 10 23 33 AM

@katiebrady-astro Adding you for review to see if there's anything I can do to optimize this on the React/ web front. I commented the line in `DocCard/index.js` that I wrote to cause this behavior. Would it be better to use `.svg` for the cloud logos, or is png alright?

**Comment by kwcanuck** [comment]
@jwitz cool stuff! As we are now using third-party logos, I would recommend the addition of an attribution statement to the footer or our docs. Here is an example:

image

The attribution statement formally recognizes ownership of trademarks and logos owned by external organizations. This could potentially save Astronomer some legal hassle.

**Comment by katiebrady-astro** [comment]
Hey @jwitz looks great! 🙌

SVGs can look sharper in multiple settings, but I think the PNGs you've got are already so small that those should work well. I also went ahead and losslessly optimized them with ImageOptim so they're _even smaller_ in size. 😄

**Pull request merged**